CHANGE-NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v3.3.0
+- Added the interactive traverse feature.
```

Comment:
I'd call it
```suggestion
- Added the semi-automatic traversal feature.
```
It's not the _interactivity_ where the traversal stands out, rather the automaticity 🤖

Reply:
I do not like the name change (traverse -> traversal) - why don't we stick to CLI nomenclature?

Reply:
Right, _traverse_ is a noun as well, let's NOT use _traversal_ then

<img width="712" alt="image" src="https://user-images.githubusercontent.com/3383210/204305966-ab0bb500-6da4-4334-b28e-160ff4452cf1.png">